tests/test_unit_modflow_dis.py

```diff
@@ -96,3 +96,87 @@ def test_get_end_time(flopy_dis, modflow_lib_path):
 
     actual_end_time = mf6.get_end_time()
     assert math.isclose(prescribed_end_time, actual_end_time)
+
+
+def test_update(flopy_dis, modflow_lib_path):
+    model_path, sim = flopy_dis
+    os.chdir(model_path)
+    mf6 = XmiWrapper(modflow_lib_path)
+
+    # Write output to screen:
+    mf6.set_int("ISTDOUTTOFILE", 0)
+
+    # Initialize
+    mf6_config_file = os.path.join(model_path, "mfsim.nam")
+    mf6.initialize(mf6_config_file)
+
+    # Advance model by single time step
+    mf6.update()
+
+
+def test_get_current_time(flopy_dis, modflow_lib_path):
+    model_path, sim = flopy_dis
+    os.chdir(model_path)
+    mf6 = XmiWrapper(modflow_lib_path)
+
+    # Write output to screen:
+    mf6.set_int("ISTDOUTTOFILE", 0)
+
+    # Initialize
+    mf6_config_file = os.path.join(model_path, "mfsim.nam")
+    mf6.initialize(mf6_config_file)
+
+    # Advance model by single time step
+    mf6.update()
+
+    # prescribed_start_time for modflow models is always 0
+    start_time = 0.0
+
+    # Get prescribed_end_time from TDIS package
+    tdis = sim.get_package("TDIS")
+    perioddata = tdis.perioddata.array
+    perlen, nstp, tsmult = perioddata[0]
+
+    if math.isclose(tsmult, 1):
+        prescribed_current_time = start_time + perlen / nstp
+    else:
+        prescribed_current_time = start_time + perlen * (tsmult - 1.0) / (
+            tsmult ** nstp - 1.0
+        )
+
+    actual_current_time = mf6.get_current_time()
+
+    assert math.isclose(prescribed_current_time, actual_current_time)
+
+
+def test_get_value_ptr(flopy_dis, modflow_lib_path):
+    """`flopy_dis` sets constant head values.
+       This test checks if these can be properly extracted."""
+
+    model_path, sim = flopy_dis
+    os.chdir(model_path)
+    mf6 = XmiWrapper(modflow_lib_path)
+
+    # Write output to screen:
+    mf6.set_int("ISTDOUTTOFILE", 0)
+
+    # Initialize
+    mf6_config_file = os.path.join(model_path, "mfsim.nam")
+    mf6.initialize(mf6_config_file)
+
+    model = sim.get_model(sim.name)
+    chd = model.get_package("chd_0")
+    stress_period_data = chd.stress_period_data.array[0]
+
+    # TODO: Refactor this as soon as `get_var_grid` is implemented
+    dis = model.get_package("dis")
+    len_column = len(dis.delr.array)
+
+    mf6.update()
+    actual_head = mf6.get_value_ptr("SLN_1/X")
```

Comment:
let's try and use <model_name>/X, which should also work now. Shortly we will be able to even use <model_name>/head (and <model_name>/concentration for transport) which is preferable I guess